### PR TITLE
add alert on pages if the current user is listed as a coauthor (#1260)

### PR DESCRIPTION
**Pull request by aspriya**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: app/views/notes/show.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <% if @node.has_power_tag('replication') %><div class="alert alert-info"><%= raw t('notes.show.replication') %> <a href="/n/<%= @node.power_tag('replication') %>"><%= raw t('notes.show.replication_link') %></a>.</div><% end %>
   <% if @node.has_power_tag('series') %><div class="alert alert-info"><%= raw t('notes.show.series') %> <a href="/tag/<%= @node.power_tag('series') %>"><%= @node.power_tag('series') %></a>.</div><% end %>
   <% if @node.has_power_tag('build') %><div class="alert alert-info"><%= raw t('notes.show.build') %> <a href="/n/<%= @node.power_tag('build') %>"><%= raw t('notes.show.build_link') %></a>.</div><% end %>
+  <% if @node.has_power_tag('with') && @node.coauthors.map {|c| c.username }.include?(current_user.username) %> <div class="alert alert-success"> You are a coauthor of this post. This attributes your contribution, and gives you edit access to the note <a href="#"> Learn more </a></div><% end %>
```
Comment (jywarren) [inline app/views/notes/show.html.erb]: Hi, this looks fantastic but can we have the link go to https://publiclab.org/wiki/power-tags#Coauthorship ?

Thank you!!!

Comment (aspriya) [inline app/views/notes/show.html.erb]: Thanks!, will make that. thanks again!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks!

**Comment by aspriya** [comment]
Wow! thanks!. Of course I will keep in touch and do some work!.

**Pull request merged**